File: src/main/kotlin/bot/BotMain.java
package bot;

import bot.kits.DiceKit;
import bot.kits.RolesKit;
import robo.Mask;

public class BotMain {

    public static void main(String[] args) {
        Mask mask = new Mask(
                new RolesKit(),
                new DiceKit()
        );
        mask.startBot(args[0]);
    }

}

File: src/main/kotlin/bot/BotMain.kt
package bot

import bot.kits.DiceKit
import bot.kits.PilotKit
import robo.Robo

object BotMain {
    @JvmStatic
    fun main(args: Array<String>) {
        val robo = Robo("robo")
        robo.installSubKits(PilotKit(), DiceKit())
        robo.startBot(args[0])
    }
}